src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -89,4 +89,20 @@ public void addMessage(Message message) {
   public void setMessages(List<Message> messages) {
     this.messages = messages;
   }
+
+  /** Returns all the messages, use with caution. Expensive! */
```

Comment:
It is probably ok not to mention that it is expensive. The method itself is not expensive but the use cases of them may be.